tests/qref/data/valid_programs/programs_with_repetitions/repetition_3_closed_form copy.yaml

```diff
@@ -0,0 +1,85 @@
+description: Program representing QPE using closed form sequence.
+input:
+  program:
+    children:
+    - name: Hadamards
+      ports:
+      - direction: through
+        name: register
+        size: N
+    - name: Evolution
+      ports:
+      - direction: input
+        name: result_in
+        size: N
+      - direction: output
+        name: result_out
+        size: N
+      - direction: input
+        name: psi_in
+        size: M
+      - direction: output
+        name: psi_out
+        size: M
+      children:
+      - name: U
+        ports:
+        - direction: through
+          name: result
+          size: N
+        - direction: through
+          name: psi
+          size: N
+        resources:
+        - name: T_gates
+          type: additive
+          value: N**2
+      connections:
+      - source: result_in
+        target: U.result
+      - source: U.result
+        target: result_out
+      - source: psi_in
+        target: U.psi
+      - source: U.psi
+        target: psi_out
+      repetition:
+        count: N
+        sequence:
+          type: closed_form
+          sum: "(2 ^ (X + 1) - 1)"
```

Comment:
**question**: In some places we use `**` to denote exponentiation, while in other places like here we use `^`. WDYT about unifying it so that every exponentiation is represented with the same symbol?

Personally, I'd like it to be `**` because it is more pythonic, and `^` is meant to mean xor in Python. We support both notations in Bartiq, but users familiar with Python might find the current state of the examples confusing - if there are examples of `**` and examples of `^`, a reasonable conclusion is that those two operations are diferent, right?

On the other hand, QREF is (programming)language-agnostic and we don't discuss language of arithmetic expressions in the documentation - it is therefore subject to interpretation by client libraries.

What's your stance on this?

Reply:
Agree, I think `**` is better choice.
It's language agnostic, but reality is that most people will use that with Python anyway, assuming pythonic syntax is not bad.